src/pyOpenMS/pyopenms/_dataframes.py

```
@@ -513,107 +514,107 @@
 
 # TODO think about the best way for such top-level function. IMHO in python, encapsulation in a stateless class in unnecessary.
 #   We should probably not just import this whole submodule without prefix.
-def peptide_identifications_to_df(peps: List[_PeptideIdentification], decode_ontology : bool = True,
+def peptide_identifications_to_df(peps: _PeptideIdentificationList, decode_ontology : bool = True,
                                   default_missing_values: dict = {bool: False, int: -9999, float: _np.nan, str: ''},
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)